ReactAndroid/src/main/java/com/facebook/react/common/DebugServerException.java

```diff
@@ -28,8 +28,8 @@
       "\u2022 Ensure that the packager server is running\n" +
       "\u2022 Ensure that your device/emulator is connected to your machine and has USB debugging enabled - run 'adb devices' to see a list of connected devices\n" +
       "\u2022 Ensure Airplane Mode is disabled\n" +
-      "\u2022 If you're on a physical device connected to the same machine, run 'adb reverse tcp:8081 tcp:8081' to forward requests from your device\n" +
-      "\u2022 If your device is on the same Wi-Fi network, set 'Debug server host & port for device' in 'Dev settings' to your machine's IP address and the port of the local dev server - e.g. 10.0.1.1:8081\n\n";
+      "\u2022 If you're on a physical device connected to the same machine, run 'adb reverse tcp:<PORT> tcp:<PORT>' to forward requests from your device\n" +
```

Comment:
Leaving a TODO for me: 
This needs access to app context to get the port, since it's now defined as a string res. Ideally this would match the current port.

Reply:
Got it. DebugServerException now accepts a URL in the `makeGeneric` method which parses out the port.